src/components/nodes/gateway/gatewayConfig.js

```diff
@@ -0,0 +1 @@
+export const gatewayDirection = { diverging: 'diverging', converging: 'converging' };
```

Comment:
It seems you export almost the same object in `src/components/nodes/gateway/index.js`:
```js
export const gatewayDirectionOptions = { Diverging: 'Diverging', Converging: 'Converging' };
```

I think this config can be removed from one of these spots.